blocks/graph.py

```diff
@@ -533,8 +533,8 @@ def collect_parameters(computation_graph, parameters):
 
 
 def apply_dropout(computation_graph, variables, drop_prob, rng=None,
-                  seed=None):
-    """Returns a graph to variables in a computational graph.
+                  seed=None, custom_divisor=None):
```

Comment:
I would place `custom_divisor` argument before `rng` and `seed` so that's easier to pass it as a not named argument.